specs/login-page.md

```
@@ -6,81 +6,362 @@ Dripnote 서비스의 모든 기능을 이용하기 위한 진입점입니다. 2
+type SocialProvider = 'google' | 'naver' | 'kakao';
 
-### 3.2 브랜드 아이덴티티 및 슬로건 (Branding)
+interface SocialButtonProps {
+  provider: SocialProvider;
+}
+```
 
-- **Logo**: 화면 상단 중앙에 **"Dripnote"** 로고(Playfair Display, Font-weight: Extrabold)를 배치합니다. 로고 하단에 적절한 여백(`mb-3`)을 두어 슬로건과의 계층 구조를 명확히 합니다.
-- **Punchy Headline**: "시작하기" (Outfit, Light, Uppercase, `tracking-[0.2em]`).
+**State**:
+| 상태명 | 타입 | 초기값 | 설명 |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)